src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -257,6 +267,14 @@
     </{{ element|default('div') }}>
 {%- endblock form_row %}
 
+{% block file_row -%}
+    <{{ element|default('div') }} class="custom-file">
```

Comment:
In your case the element is the div, maybe use <div> directly in place of element ? And you should add a `<div class="input-group mb-3">` or a simple `form-group` or at least the `mb-3` to add a margin after the row.

Reply:
Doesn't custom-file-label add a margin-bottom? (.5rem to be exactly)

Reply:
I've try and it's collapsed. In the bootstrap doc, they encapsulate the custom-file-label in an other div with a mb-3 to add the margin bottom.

Reply:
You need to add a `div` with class "form-group" around this.